src/drivers/firestore.ts

```diff
@@ -45,35 +49,33 @@ export async function initializeFirebase() {
   }
 }
 
-const findProviderId = (identity: Identity) => {
-  switch (identity.org.ssoProvider) {
+const findProviderId = (org: OrgData) => {
+  switch (org.ssoProvider) {
     case "google":
-      return SignInMethod.GOOGLE;
+      return ProviderId.GOOGLE;
```

Comment:
Changed this to a providerId constant. The value of the constant is unchanged: "google.com".